tests_integration/__tests__/piped-output.js

```diff
@@ -0,0 +1,50 @@
+"use strict";
+
+const runPrettier = require("../runPrettier");
+
+test("output with --list-different + unformated differs when piped", () => {
```

Comment:
Typo: "formated" → "formatted"

There are more instances of this below, including some file names.

Reply:
I've copied the texts from the `write.js` tests. Shall I rename the files in there as well or do you want to just have the string typos fixed and the file changes in another PR?

Reply:
I see. Let's do it in another PR (if you feel like it).